### ISSUE TITLE
Improve error message when using "set-current" while not logged in

### ISSUE THREAD

**Issue by lumaxis** [microsoft/appcenter-cli#540]
## Description

When trying to use `set-current` in a logged out state, the CLI results in an unhelpful TypeError:
```bash
Command Failure at TypeError: Cannot set property 'defaultApp' of null
    at SetCurrentAppCommand.<anonymous> (/usr/local/lib/node_modules/appcenter-cli/dist/commands/apps/set-current.js:37:32)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/appcenter-cli/dist/commands/apps/set-current.js:10:58)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
Error: Command 'apps set-current lumaxis/Test --debug --token my-token' failed with exception "Cannot set property 'defaultApp' of null"
```

This happens because `profile` in https://github.com/Microsoft/appcenter-cli/blob/6f869abf5f03b07ca85f45e1e65df4a03507298a/src/commands/apps/set-current.ts#L33 is `null` since there is no currently logged-in user.

## Repro steps

```bash 
appcenter logout
appcenter apps set-current 'lumaxis/Test' --token $TOKEN`
```

**Comment by jp-andre** [comment]
I think this is fixed by #742 where the error message has been changed.

**Comment by lumaxis** [comment]
I’m not sure it was #712 but it’s definitely fixed now!
